openassessment/xblock/static/js/src/lms/oa_staff_area.js

```diff
@@ -157,108 +198,148 @@
+         *
+         * Check whether the cancel submission button is enabled.
+         *
+         * Args:
+         *   enabled (bool): If specified, set the state of the button.
+         *
+         * Returns:
+         *   bool: Whether the button is enabled.
+         *
+         * Examples:
+         * >> view.submitEnabled(true);  // enable the button
+         * >> view.submitEnabled();  // check whether the button is enabled
+         * >> true
+         */
         cancelSubmissionEnabled: function(enabled) {
-            var sel = $('#submit_cancel_submission', this.element);
+            var $cancelButton = $('.action--submit-cancel-submission', this.element);
             if (typeof enabled === 'undefined') {
-                return !sel.hasClass('is--disabled');
+                return !$cancelButton.hasClass('is--disabled');
             } else {
-                sel.toggleClass('is--disabled', !enabled);
+                $cancelButton.toggleClass('is--disabled', !enabled);
             }
         },
 
         /**
-         Set the comment text.
-         Retrieve the comment text.
-
-         Args:
-         text (string): reason to .
-
-         Returns:
-         string: The current comment text.
-         **/
+         * Set the comment text.
+         *
+         * Retrieve the comment text.
+         *
+         * Args:
+         *   text (string): reason to .
+         *
+         * Returns:
+         *   string: The current comment text.
+         */
         comment: function(text) {
-            var sel = $('#staff-info__cancel-submission__comments', this.element);
+            var $submissionComments = $('.cancel_submission_comments', this.element);
             if (typeof text === 'undefined') {
-                return sel.val();
+                return $submissionComments.val();
             } else {
-                sel.val(text);
+                $submissionComments.val(text);
             }
         },
 
         /**
-         Enable/disable the cancel submission based on whether
-         the user has entered a comment.
-         **/
+         * Enable/disable the cancel submission based on whether
+         * the user has entered a comment.
+         */
         handleCommentChanged: function() {
             // Enable the cancel submission button only for non-blank comments
-            var isBlank = ($.trim(this.comment()) !== '');
+            var isBlank = $.trim(this.comment()) !== '';
             this.cancelSubmissionEnabled(isBlank);
+        },
+
+
+        /**
+         * Enable/disable the staff assessment submit button.
+         *
+         * @param enabled If specified, sets the state of the button.
+         * @returns {boolean} Whether the button is enabled
+         */
+        staffSubmitEnabled: function(enabled) {
+            var button = $('.wrapper--staff-assessment .action--submit', this.element);
+            if (typeof enabled === 'undefined') {
```

Comment:
This is a confusing method. It does two different things depending on whether enabled is undefined-- it either returns the state of the button (which is what is documented), or it toggles the button state (and has no explicit return). At the very least, it should always return the state of the button (I would lean towards 2 different method).